pitch.md

```diff
@@ -23,38 +89,174 @@ should be more specific than "The user describes properties of unicorns.", e.g.,
+as I can tell).  I am also looking at some of the features provided for
+[Premium Users](http://whenisgood.net/PremiumInfo).
+
+My solution to the scheduling problem might be a bit more
+ambitious in the functionality it hopes to provide.  This does allow for more
+expressiveness, which is nice, but expressiveness is a double edged sword,
+I will need to take care during design to avoid feature creep and extra complexity
+for users.  Perhaps I will find that the existing websites' simplicity is
+more desirable than the benefits of the expressiveness I aim to provide (but
+I hope not!).
+
+I think that they address the need pretty well at the present, but I think that
+my DSL could cut down on repetition and allow users to better express themselves.
 
 ## Suitability
 _What percentage of your time do you think will be spent directly engaging in
 the **language** aspects of this project (e.g., making language design
 decisions), as opposed to "systems" aspects of the project (e.g., implementing a
 complicated semantics that doesn't require a lot of language design)?_
 
+The main reason I finally settled on this project was that I felt it had the best
+balance of **language** design to systems programming of any of my potential
+choices.  As I mentioned in the previous assignment, the underlying internal
+representation could be represented with just a [protocol
+buffer](https://en.wikipedia.org/wiki/Protocol_Buffers) for each individual.
+
+Armed with the parsing tools from the Picobot lab, the work of writing a 
+parser to take the syntax of my language to the internal representation should 
+be overshadowed by the work of designing the best interface/fluency of the
+language for the user's sake.
+
+Based on Daniel Sonner's helpful advice, I have decided not to focus on the
+algorithms to find an optimal schedule given a set of constraints, which could
+potentially be *very* interesting, so that I can focus the lion's share of
+my time on designing an intuitive syntax and then implementing it.
 
 ## Scope
 _How big an idea is this? How ambitious is this project?_
 
+I think that this is a good, ambitious project.  It has enough extensions and
+places to cut back such that I will be able to make a very **very** minimal
+prototype upon which I can iterate, gradually adding more of the
+desired characteristics.
+
+This will ensure that it isn't to big to get off the ground, but that there
+are enough extensions to the project to keep me busy.
 
 ## Challenges and opportunities
 _Why is this a good project? What are some challenges you expect to face? How
 might you overcome them? What are some warning signs that the project has gotten
 off track, and how will you get the project back on track if needed?_
+
+I think it is a good idea, since it has a use case that individuals do encounter
+somewhat regularly so this DSL might just be able to fit into that niche.
+
+My worries that the idea was more of an "app" than a "language" have been
```

Comment:
I think this is definitely more of an language. There could be a slick app to go along with it, but the focus should remain on the language.